src/NuGet.Clients/NuGet.SolutionRestoreManager/SolutionRestoreJob.cs

```diff
@@ -499,6 +507,23 @@ private async Task RestoreAsync(bool forceRestore, RestoreOperationSource restor
                                 {
                                     if (isRestoreSucceeded)
                                     {
+                                        foreach (RestoreSummary summary in restoreSummaries)
+                                        {
+                                            foreach (IRestoreLogMessage error in summary.Errors)
+                                            {
+                                                if (error.Code.Equals(NuGetLogCode.NU1901) || error.Code.Equals(NuGetLogCode.NU1902) || error.Code.Equals(NuGetLogCode.NU1903) || error.Code.Equals(NuGetLogCode.NU1904))
```

Comment:
Maybe it's time for a utility method to just flag when a restore error had any vulnerability codes within it?

Reply:
Moved this to a `LogUtility.cs`, I wasn't sure if we should create a new file, this is the one made more sense to me.